### ISSUE TITLE
TypeError: _this._element is undefined on close animation in progress/kendo-react-dropdowns 0.3.0

### ISSUE THREAD

**Issue by georgeri** [telerik/kendo-react#7]
## I'm submitting a

* Bug report 

## Current behavior
progress/kendo-react-dropdowns 0.3.0.
When multiple DropDownList components with filtering are on the same page, clicking on next dropdownlist with in a second causes an un-handled error "TypeError: _this._element is undefined"
I think it is happening if the second dropdown is clicked while the close animation of the first one is not yet complete.
TypeError: _this._element is undefined
```
Popup/_this.onClosing
Popup.js:77
<anonymous> self-hosted:950:17 onExiting
AnimationChild.js:93
CSSTransition/_this.onExiting
CSSTransition.js:252
performExit/<
Transition.js:273
setNextCallback/this.nextCallback
Transition.js:318
safeSetState/<
Transition.js:304
commitCallbacks
react-dom.development.js:6163
commitLifeCycles
react-dom.development.js:8784
commitAllLifeCycles
react-dom.development.js:9946
callCallback
react-dom.development.js:542
invokeGuardedCallbackDev
react-dom.development.js:581
invokeGuardedCallback
react-dom.development.js:438
commitRoot
react-dom.development.js:10050
performWorkOnRoot
react-dom.development.js:11017
performWork
react-dom.development.js:10967
requestWork
react-dom.development.js:10878
scheduleWorkImpl
react-dom.development.js:10732
scheduleWork
react-dom.development.js:10689
enqueueSetState
react-dom.development.js:6212
./node_modules/react/cjs/react.development.js/Component.prototype.setState
react.development.js:237
safeSetState
Transition.js:302
performExit/</<
Transition.js:276
setNextCallback/this.nextCallback
````

## Expected behavior
Clicking on dropdownlist to see options should work without error when there is multiple dropdownlist components on screen.

## Minimal reproduction of the problem with instructions

https://plnkr.co/edit/VQCFGtAXKbLeaqjriKLq?p=preview

## What is the motivation or use case for changing the behavior?

No change in behaviour - fix the error.
## Environment
React 16. 
Package versions:
progress/kendo-data-query: 1.2.0,
progress/kendo-react-dropdowns": 0.3.0,
progress/kendo-react-layout: 0.3.0,
progress/kendo-ui: 2018.1.117,

```

Browser:
<!-- Leave only the browsers that you tested the issue with. -->
- Chrome (desktop) Version 64.0.3282.167 (Official Build) (64-bit)
- Firefox version 58.0.1 (64-bit)

System:
- Node version: v6.11.5, npm: 3.10.10
- Platform:  Windows

<!-- Add any additional information that might be relevant. -->

**Comment by mitagg** [comment]
@georgeri - we have published a fix in our development channel, under a @dev tag. Please refer to the following [article](https://www.telerik.com/kendo-react-ui/components/installation/development-builds/) on how to obtain the development builds.

**Comment by nstoychev** [comment]
Fixed in official [v0.4.0](https://www.telerik.com/kendo-react-ui/components/changelogs/ui-for-react/),  already works in the example above since it uses `@latest` version.
